doc/introduction/data.rst

```diff
@@ -89,6 +89,8 @@ been generated for formic acid at zero temperature. The GBS samples can be used
 
 .. autosummary::
     Formic
+    Water
+    Pyrrole

```

Comment:
Water and Pyrrole have been added, but it looks like `PHat`, `MUTAG`, `QM9Exact`, and `QM9MC` are still missing?

Reply:
@soranjh @ixfoduap @trbromley

Reply:
Thanks @josh146! Added feature-vector based ones in e8a69683634553eb513a29946ee5ceed7dde1be4 and PHat in 5ca3df6d40a0ed9fc8ab75c71d11c33396413c65.